unlock-app/public/template.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Unlock</title>
+  <link rel="stylesheet" href="template.css">
```

Comment:
Should we be calling the stylesheet something else? I'm not sure how descriptive `template` is. But it's probably fine for now.